### ISSUE TITLE
TF14045: The identity is not a recognized identity. ##[error]Exit code 100 returned from process

### ISSUE THREAD

**Issue by toniellenk** [Microsoft/vsts-agent#1761]
I've done the migration from TFS 2015 to TFS 2018, everything is working correctly, but the Build Agent is experiencing an error while giving Get Source

## Have you ever tried to solve problems?

- **Reinstall the agent pool;**
- **Create a new Build Definitions;**
- **Reinstall the VSTS agent service;**
- **I have identified that the identity "781ab373-f96f-457f-9f86-fbea51914f25" belongs to a particular TFS user;**

## Agent Version and Platform

**Current agent version: '2.122.2'
S.O: Windows Server 2012 R2 Standart**

## Type and Version of VSTS
**TFS 2018 SP 1**

## What is not working?
**Job Get sources**

![image](https://user-images.githubusercontent.com/11699501/43998807-92e8b5fc-9dcc-11e8-9cf8-22725a62b071.png)

## agent and worker diagnostic logs

[logs_38091.zip](https://github.com/Microsoft/vsts-agent/files/2280741/logs_38091.zip)

**Comment by TingluoHuang** [comment]
@toniellenk if you check your TFS AT's event log, Applications->TeamFoundationServer -> debug, find the time that this error happened, you might able to find some useful error on the server side.

**Comment by toniellenk** [comment]
@TingluoHuang thanks for the help, but I've already been able to solve the problem.

I have identified that there were some workspaces linked to this user in the tbl_Workspace table of the TFS_Collection database. After removing those user records in the table the problem has been resolved.

I used the following script to identify:

select * from tbl_Workspace where OwnerId = '781ab373-f96f-457f-9f86-fbea51914f25'
